projects/packages/videopress/src/client/state/actions.js

```diff
@@ -74,7 +80,7 @@ const updateVideoPrivacy = ( id, level ) => async ( { dispatch } ) => {
 
 	// Let's be optimistic and update the UI right away.
 	// @todo: Add a loading state to the state/UI.
-	dispatch.setVideo( {
+	dispatch.setVideoPrivacy( {
```

Comment:
The `@todo` above is addressed in this PR

Reply:
let me remove it in a follow up. I'd wouldn't like to restart the tests just for this.